tuf-spec.md

```diff
@@ -1285,12 +1289,15 @@ it in the next step.
   example, Y may be 2^10.
 
 4. **Check for an arbitrary software attack.** Version N+1 of the root
-  metadata file MUST have been signed by: (1) a threshold of keys specified in
-  the trusted root metadata file (version N), and (2) a threshold of keys
-  specified in the new root metadata file being validated (version N+1).  If
-  version N+1 is not signed as required, discard it, abort the update cycle,
-  and report the signature failure.  On the next update cycle, begin at step
-  [[#update-root]] and version N of the root metadata file.
+  metadata file MUST have been signed by: (1) a <a>THRESHOLD</a> of keys
+  specified in the trusted root metadata file (version N), and (2) a
+  <a>THRESHOLD</a> of keys specified in the new root metadata file being
+  validated (version N+1).  When computing the <a>THRESHOLD</a> each
+  <a>KEY</a> must ONLY contribute one <a>SIGNATURE</a> towards the
```

Comment:
Right, but what does this mean in practice? Checking only for duplicate keyids, or making sure that differently encoded keys with the same parameters don't slip by? Needs to be clearer.

Reply:
What do you have in mind here? With the info we have in TUF metadata we can easily check for duplicate keyids. Would could recommend (require?) implementations also check that the same `keyval.public` isn't used by multiple contributing keys?

Reply:
I've tried to be much more explicit with some edits in fe15850